Formula/math-comp.rb

```diff
@@ -0,0 +1,46 @@
+class MathComp < Formula
+  desc "Mathematical Components for the Coq proof assistant"
+  homepage "https://math-comp.github.io/math-comp/"
+  url "https://github.com/math-comp/math-comp/archive/mathcomp-1.6.4.tar.gz"
+  sha256 "c672a4237f708b5f03f1feed9de37f98ef5c331819047e1f71b5762dcd92b262"
+  head "https://github.com/math-comp/math-comp.git"
+
+  option "without-docs", "Don't generate or install documentation"
+
+  depends_on "ocaml"
+  depends_on "coq"
+
+  def install
+    ENV["COQBIN"] = "#{Formula["coq"].bin}/"
+    (buildpath/"mathcomp/Makefile.coq.local").write(<<~EOF)
+      COQLIB=#{lib}/coq/
+    EOF
+
+    cd "mathcomp" do
+      system "make", "MAKEFLAGS=#{ENV["MAKEFLAGS"]}"
```

Comment:
It should pick the makeflags up from the environment automatically. Also, can we not just pass `COQBIN` and `COQLIB` on this line?

Reply:
MAKEFLAGS from the environment is not respected because the Makefile defines MAKEFILE variable. It must be specified in the command line to override the definition.

Reply:
ok.

Reply:
Changed `COQBIN` to be passed on the command line.
`COQLIB` must be specified in the file.

Reply:
ok.